src/EditorFeatures/TestUtilities2/Intellisense/ITestState.vb

```diff
@@ -144,12 +144,12 @@ Namespace Microsoft.CodeAnalysis.Editor.UnitTests.IntelliSense
 
         Function WaitForAsynchronousOperationsAsync() As Task
 
-
         Function AssertSelectedCompletionItem(
                                Optional displayText As String = Nothing,
                                Optional description As String = Nothing,
                                Optional isSoftSelected As Boolean? = Nothing,
                                Optional isHardSelected As Boolean? = Nothing,
+                               Optional displayTextSuffix As String = Nothing,
```

Comment:
>Optional displayTextSuffix As String = Nothing, [](start = 31, length = 47)

put it closer in order to displayText?

Reply:
that woudl require updating a *huge* amount of tests taht currently do not pass along displayTextSuffix, but do pass along display/description.  i'd prefer to not do that :)